tests/ui/manual_assert.rs

```diff
@@ -1,6 +1,6 @@
 // revisions: edition2018 edition2021
-// [edition2018] edition:2018
-// [edition2021] edition:2021
+//[edition2018] edition:2018
+//[edition2021] edition:2021
```

Comment:
I'm curious: Does this compile the code under both revisions?

Reply:
It does yeah, a test is compiled once per revision, the `[revision]` syntax makes the magic comment thingies only apply on that revision